build/template-desktop-unit-and-automation.yaml

```diff
@@ -6,9 +6,9 @@ steps:
     version: 3.x
 
 - task: UseDotNet@2
-  displayName: 'Use .Net Core SDK 6.x'
+  displayName: 'Use .Net Core SDK 5.x'
   inputs:
-    version: 6.x
+    version: 5.x
```

Comment:
5.x is already deprecated. Can we stay on 6?

Reply:
https://dotnet.microsoft.com/en-us/download/dotnet/5.0

Reply:
Switching to 6.x causes failure in the desktop tests in NET5-WIN configuration.
##[error]Testhost process exited with error: It was not possible to find any compatible framework version
##[error]The framework 'Microsoft.WindowsDesktop.App', version '5.0.0' (x64) was not found.
##[error]  - The following frameworks were found: